src/agents/read_agent.py

```
@@ -1,51 +1,143 @@
+        message_lower = message.lower()
+        
+        # Check for field keywords
+        if any(keyword in message_lower for keyword in field_keywords):
+            return True
+            
+        # Check for actual field names from database (common field names)
+        common_field_names = ["橋前", "橋向こう", "登山道前", "豊緑", "toyomidori"]
+        if any(field_name in message_lower for field_name in common_field_names):
+            return True
             
+        return False
```

Ruff (0.12.2)

Return the condition directly

Inline condition

(SIM103)